### PR TITLE
Removed vertical tab flyout

### PR DESCRIPTION
**Pull request by yaichenbaum**
**Details of Changes**
Add details of changes here.
Removed the vertical tab flyout since it was rarely used

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Review by gave92 — APPROVED** [review]

File: src/Files.Uwp/Views/MainPage.xaml [hunk 1]
```diff
@@ -1,486 +1,450 @@
...
-    <Page.Resources>
-        <ResourceDictionary>
-            <ResourceDictionary.MergedDictionaries>
-                <ResourceDictionary Source="ms-appx:///ResourceDictionaries/DefaultGridSplitterStyle.xaml" />
-            </ResourceDictionary.MergedDictionaries>
+	x:Class="Files.Uwp.Views.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:contract8Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,8)"
+	xmlns:controls="using:Files.Uwp.UserControls"
```
Comment (gave92) [inline src/Files.Uwp/Views/MainPage.xaml:6]: Spaces turned to tabs intentional?

Comment (yaichenbaum) [reply src/Files.Uwp/Views/MainPage.xaml:6]: Yeah, btw, you can hide whitespaces in GitHub to make it easier to review.

Comment (gave92) [reply src/Files.Uwp/Views/MainPage.xaml:6]: This `?w=1`? Yup been happier since I found out :)

Comment (yaichenbaum) [reply src/Files.Uwp/Views/MainPage.xaml:6]: I think there is also a setting for it but that works

**Pull request merged**